src/wazuh_db/schema_global.sql

```diff
@@ -6,6 +6,8 @@
  * and/or modify it under the terms of GPLv2.
  */
 
+PRAGMA journal_mode=DELETE;
```

Comment:
Is this necessary here? As per the discussion that I had with @vikman90 , this is necessary only on the upgrade.

Reply:
This is true, the default mode is **DELETE** 
[https://sqlite.org/wal.html](https://sqlite.org/wal.html)